app/backend/src/clj/teet/authorization/authorization_queries.clj

```diff
@@ -0,0 +1,11 @@
+(ns teet.authorization.authorization-queries
+  "Fetch enumeration values"
+  (:require [teet.db-api.core :as db-api :refer [defquery]]))
+
+
+(defquery :authorization/permissions
+  {:doc "Fetch required permissions for commands and queries"
+   :args {}
+   :project-id nil
+   :authorization {}}
```

Comment:
should this allow unauthenticated use?

Reply:
I don't currently see a use case for that.

Reply:
perhaps not currently, but we will have to support guest use

Reply:
I can add the unauthenticated? true.